### PR TITLE
Added python environment for regression.py.

### PR DESCRIPTION
**Pull request by balhoff**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -21,12 +21,24 @@ RUN apt-get update && apt-get upgrade -y \
     make \
     curl \
     wget \
-    tar
+    tar \
+    python2.7 python2.7-dev python-pip
```
Comment (hlapp) [inline Dockerfile:25]: Each package on a line by itself, please? Also, doesn't `-dev` include `python2.7`?

File: Dockerfile [hunk 2]
```diff
@@ -55,6 +67,6 @@ ENV PATH "/tools/blazegraph-runner-$BGR/bin:$PATH"
 ###### KB-OWL-TOOLS ######
 RUN curl -O -L https://github.com/phenoscape/phenoscape-owl-tools/releases/download/v$KBOT/kb-owl-tools-$KBOT.tgz \
     && tar -zxf kb-owl-tools-$KBOT.tgz \
-    && chmod +x /tools/kb-owl-tools-$KBOT
+    && chmod +x /tools/kb-owl-tools-$KBOT && curl -L -O https://github.com/phenoscape/phenoscape-owl-tools/raw/v$KBOT/src/regression.py
```
Comment (hlapp) [inline Dockerfile:70]: Second command on a line by itself, please.

File: Dockerfile [hunk 3]
```diff
@@ -21,12 +21,24 @@ RUN apt-get update && apt-get upgrade -y \
...
 
+
+### Python packages ###
+RUN pip2 install virtualenv &&\
+    virtualenv -p python2.7 pyenv &&\
+    . pyenv/bin/activate &&\
+    pip2 install 'pandas==0.19.1' &&\
+    pip2 install 'scipy==0.15.1' &&\
+    pip2 install 'patsy==0.4.1' &&\
+    pip2 install 'statsmodels==0.6.1'
```
Comment (hlapp) [inline Dockerfile:39]: Shouldn't this be in a `requirements.txt` file?

File: Dockerfile (hunk 1 above)
Comment (balhoff) [reply Dockerfile:25]: I don't mind putting them on multiple lines, but I did this on purpose. I see these as a package deal. I would suppose `-dev` includes `python2.7`. The effect of relying on transitivity is not quite the same with regards to uninstalling, although that will never happen in this environment.

File: Dockerfile (hunk 3 above)
Comment (balhoff) [reply Dockerfile:39]: Possibly, but getting a working set of these packages and versions involved quite a lot of anguish. I wanted it to happen in exactly the order that I finally got it to work. I guess a `requirements.txt` with the versions fixed would have the same result, but these things are really touchy.

Comment (balhoff) [reply Dockerfile:41]: You know I think I prefer it this way anyway; it makes the docker file a little more self-contained and at the moment I don't expect this list to grow.

**Comment by balhoff** [comment]
Thanks, I made these changes.

**Pull request merged**